controller/src/main/java/io/pravega/controller/server/bucket/BucketManager.java

```diff
@@ -62,28 +61,36 @@ protected void doStart() {
     }
 
     private CompletableFuture<Void> tryTakeOwnership(int bucket) {
-        return RetryHelper.withIndefiniteRetriesAsync(() -> streamMetadataStore.takeBucketOwnership(bucket, processId, executor),
-                e -> log.warn("exception while attempting to take ownership"), executor)
+        return RetryHelper.withIndefiniteRetriesAsync(() -> bucketStore.takeBucketOwnership(serviceType, bucket, 
+                processId, executor),
+                e -> log.warn("exception while attempting to take ownership for bucket {} for service {}", bucket, serviceType), executor)
```

Comment:
Maybe log `e.getMessage` or `.getClass` to indicate what kind of exception is logged (which IMO, is better than not indicating at all or recording the whole stack trace)